game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -28,7 +28,7 @@
   private String gameName = "";
   private String gameVersion = "";
   private String gameRound = "";
-  private String fileName = "";
+  private String fileName;
```

Comment:
Be careful here.  The constructor calls `setGameData()`, which invokes callbacks, which may call back into this object before `fileName` gets set to the empty string after `setGameData()` returns.